dev/Repeater/APITests/Common/DataAsElementElementFactory.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using Windows.UI.Xaml.Controls;
+
+namespace Windows.UI.Xaml.Tests.MUXControls.ApiTests.RepeaterTests.Common
+{
+    using Microsoft.UI.Xaml.Controls;
+    using ElementFactory = Microsoft.UI.Xaml.Controls.ElementFactory;
+
+    class DataAsElementElementFactory : ElementFactory
```

Comment:
This would be the default behavior if ItemTemplate on Repeater is not set at all right ?

Reply:
yes, but items repeater changes its logic if a factory is present, even if it doesn't do anything.

---
In reply to: [355563295](https://github.com/microsoft/microsoft-ui-xaml/pull/1740#discussion_r355563295) [](ancestors = 355563295)